liquibase-maven-plugin/src/main/java/org/liquibase/maven/plugins/LiquibaseDropAll.java

```diff
@@ -43,9 +36,9 @@ protected void performLiquibaseTask(Liquibase liquibase) throws LiquibaseExcepti
             for (String name : schemas.split(",")) {
                 schemaObjs.add(new CatalogAndSchema(catalog, name));
             }
-            liquibase.dropAll(dropDbclhistory, schemaObjs.toArray(new CatalogAndSchema[0]));
```

Comment:
Quick question, I know `dropDbclhistory` is PRO mvn parameter, but considering we are updating this logic instead of removing it, shouldn't we create a new one with the updates you made and deprecate the one we want to get rid of?

Reply:
The pro-logic still exists in the new commercial-maven-plugin: https://github.com/liquibase/liquibase-pro/tree/master/liquibase-commercial-maven-plugin  . So the ones using pro license need to migrate to it (that in fact is a copy of this one, just changing `org.liquibase` by `com.liquibase`) .